### ISSUE TITLE
All page content MUST be contained by landmarks - div.veteran-banner

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-cms#14307]
## Description

On Benefits Hub landing pages (i.e [Health care](https://www.va.gov/health-care/) or [Decision reviews](https://www.va.gov/decision-reviews/)), the div that contains the veteran portraits is not contained by a landmark - `<div class="veteran-banner">`. All page content **must** be contained by landmarks. 

## Steps to Recreate

1. Go to `https://www.va.gov/health-care/` in browser
2. Run axe scan using browser tools
3. Verify that the div.veteran-banner is identified as not contained by landmarks.

## Acceptance Criteria

As a screen reader user, I want to navigate the content of the page by landmark regions.

## Possible Fixes (optional)

This requires a page template change, to enclose this div.veteran-banner within `<main>`

The HTML should look like this:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/106678594/5a85c42c-e3f5-4e36-b6ec-0baeab79d825)

### Why it matters

Screen reader users can navigate to a section based on its HTML element or ARIA Landmark. It is best practice to contain all content excepting skip links, within distinct regions such as the header, nav, main, and footer. As this content is relatively decorative, it is seems less urgent. However, since screen reader users can't tell what is and isn't decorative, containing content within landmarks as the rule describes will prevent confusion.

Navigating a web page is far simpler for screen reader users if the content splits between multiple high-level sections. Content outside of sections is difficult to find, and the content's purpose may be unclear.

Historically, HTML lacked some key semantic markers such as the ability to designate sections of the page as the header, navigation, main content, and footer. Using both HTML5 elements and ARIA landmarks in the same element is considered a best practice, but the future favors using native HTML5 element regions as browser support increases.

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - All page content must be contained by landmarks](https://dequeuniversity.com/rules/axe/3.4/region?application=AxeChrome)

## Screenshots or Trace Logs

![vsapublicw-ama-landmarks](https://user-images.githubusercontent.com/57469/75595640-3cfda100-5a5b-11ea-9124-5d0aa3771d92.png)

## Acceptance criteria

- [x] Veteran banner above footer is contained within `main`
- [x] A11y review for Screen reader behavior

**Comment by laflannery** [comment]
@randimays The issue is still there, I think it might be your settings. Can you make sure that you have "Best Practices" turned on and you are using 2.1 AA?

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/a6949e3e-7ea2-460a-97c2-c3975cc79d3b)

**Comment by randimays** [comment]
Thanks @laflannery that fixed it!

**Comment by randimays** [comment]
Validated in production. Closing!
